qemu/tests/slof_device_tree.py

```diff
@@ -0,0 +1,80 @@
+import logging
+
+from autotest.client.shared import utils, error
+from virttest import utils_misc
+from virttest.qemu_devices import qcontainer
+from avocado.core import exceptions
+
+
+@error.context_aware
+def run(test, params, env):
+    """
+    slof_device_tree test:
+    steps:
+    1. Boot up a guest.
+    2. Check the guest device tree information
+    3. Compare the value with expected result.
+    :param test: Kvm test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+
+    def get_info(vm_session, guest_info):
+        """
+        Check the corresponding information from vm.
+
+        :param vm_session: session to checked vm.
+        :return: corresponding prompt
+        """
+        (status, output) = vm_session.cmd_status_output(
+            "echo `cat /proc/device-tree/%s`" % guest_info)
```

Comment:
Because if no "echo", the output will be like:
'17d7eb2c-8270-4222-bb1f-28d6c08d62db\x00[root@dhcp71-80 ~]#